templates/advantage/index.html

```diff
@@ -39,257 +33,264 @@ <h1>Ubuntu Advantage for&nbsp;Infrastructure</h1>
-              <button class="u-toggle" aria-controls="#expanded-row-cc-eal-{{ outer_loop.index }}-{{ loop.index }}" aria-expanded="false" data-shown-text="Hide" data-hidden-text="Show">
-                <span class="u-truncate"><i class="p-icon--success"></i> &nbsp;<i class="p-icon--contextual-menu">Open</i></span>
-              </button>
-              {% else %}
-              <span style="padding-top:.35rem;">N/A</span>
-              {% endif %}
-            </td>
-
-            <td id="expanded-details-{{ outer_loop.index }}-{{ loop.index }}" class="p-table-expanding__panel" aria-hidden="true">
-              <div class="row">
-                <div class="col-12 u-align--center">
-                  <table class="p-table--advantage-contract">
-                    <tr>
-                      <td class="u-align--right">Created on:</td>
-                      <td class="u-align--left"><strong>{{ contract['contractInfo']['createdAtFormatted'] }}</strong></td>
-                    </tr>
-                    {% if contract['contractInfo']['effectiveTo'] %}
-                    <tr>
-                      <td class="u-align--right">Effective to:</dt>
-                      <td class="u-align--left"><strong>{{ contract['contractInfo']['effectiveToFormatted'] }}</strong></td>
-                    </tr>
+    <section class="p-strip--light is-shallow u-hide p-card u-no-margin--bottom" aria-hidden="true" id="enterprise-table" style="border-left: 0; border-right: 0; overflow: visible;">
+      <div class="row u-arrow-up">
+        <div class="col-12">
+          {% include "advantage/_table.html" %}
+        </div>
+      </div>
+    </section>
+
+    <section class="p-strip u-no-padding--top is-shallow">
+      <div class="row">
+        <div class="col-12">
+          {% for account in enterprise_contracts %}
+            {% set outer_loop = loop %}
+            <table class="p-table-expanding p-table--advantage">
+              <thead>
+                <tr class="p-table__row">
+                  <th>{{ account }}</th>
+                  <th>UA plan</th>
+                  <th class="u-align--center">Attached</th>
+                  <th class="u-align--center">ESM</th>
+                  <th class="u-align--center">Livepatch</th>
+                  <th class="u-align--center">FIPS</th>
+                  <th class="u-align--center">CC-EAL</th>
+                  <th class="u-hide">&nbsp;</th>
+                  <th class="u-hide">&nbsp;</th>
+                  <th class="u-hide">&nbsp;</th>
+                  <th class="u-hide">&nbsp;</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for contract in enterprise_contracts[account] %}
+                  {% if contract["contractInfo"]["status"] != 'expired' %}
+
+                    {% if contract["contractInfo"]["daysTillExpiry"] %}
+                      {% set expiring_soon = contract["contractInfo"]["daysTillExpiry"] <= 3000 %}
```

Comment:
I would suggest moving this logic to the python side and adding a status of `expiring`